tests/test_jax_import_hardening.py

```
@@ -0,0 +1,53 @@
+        try:
+            _ = stormlog.jax.JAXMemoryProfiler
+        except ImportError as exc:
+            assert "stormlog[jax]" in str(exc)
+        else:
+            raise AssertionError("Expected JAXMemoryProfiler symbol load to fail lazily without jax")
+
+        print("ok")
+        """
+    )
+
+    completed = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)